src/components/MoonCard.js

```diff
@@ -1,59 +1,64 @@
-import React, { useState } from 'react'
-import { Link, useRouteMatch } from 'react-router-dom'
-import PropTypes from 'prop-types' 
-import { nanoid } from 'nanoid'
-import { createCover } from '../utility/util'
+import { useContext, useState } from 'react'
+import { useLocation } from 'react-router-dom'
+import { useNavigate } from 'react-router-dom'
+import { GameContext } from '../store/GameContext'
+import { randomCover } from '../utility/util'
 
+export const MoonCard = ({ card }) => {
+	const { state: { creationCard, gameStarted } } = useContext(GameContext)
+	const [isFlipped, setFlipped] = useState(false)
+	const [clicks, setClicks] = useState(0)
```

Comment:
I believe this bit of state needs to be lifted up a component. actions up, data down like. Think about it, a moon card can only be clicked/flipped once, but the GamePage needs to keep track of how many cards have been flipped.

If you add some console.logs you'll see that clicks never increases.

Reply:
# !!! 

# wowowow

Amazing catch! I was looking at the URL and seeing it update that it never occurred to me that I needed to observe the clicks!